backend/kernelCI_app/management/commands/notifications.py

```diff
@@ -423,23 +423,32 @@ def evaluate_test_results(
     for test in tests:
         platform = test.get("platform", "unknown platform")
         config_name = test.get("config_name", "unknown config")
-        grouped[platform][config_name][test["path"]].append(test)
+        arch = test.get("arch", "unknown architecture")
```

Comment:
The query returns an `architecture` value, not `arch`
```suggestion
        arch = test.get("architecture", "unknown architecture")
```

Reply:
Fixed it.